### PR TITLE
Add Support SCE_GXM_TEXTURE_FORMAT_U5U6U5_BGR

### PR DESCRIPTION
**Pull request by Zangetsu38**
-fix correct value return for U5U6U5_RGB.
-Add log for indicate if value is missing (i have founded this fix like that)

Fix NumptyPhysics ingame, Move him to playable

### CODE REVIEW

File: src/emulator/gxm/src/gxm.cpp [hunk 1]
```diff
@@ -759,6 +764,7 @@ GLenum translate_type(SceGxmTextureFormat format)
             return GL_UNSIGNED_SHORT_5_6_5;
         default:
         {
+            LOG_WARN("Unsupported translate type: {:#08X}", base_format);
```
Comment (Rinnegatamante) [inline src/emulator/gxm/src/gxm.cpp]: Shouldn't this be `format`?

Comment (Zangetsu38) [reply src/emulator/gxm/src/gxm.cpp]: oups confonded bnoth lol

File: src/emulator/gxm/src/gxm.cpp [hunk 2]
```diff
@@ -699,6 +703,7 @@ GLenum translate_format(SceGxmTextureFormat src) {
             return GL_INTENSITY;
         default:
         {
+            LOG_WARN("Unsupported translate format: {:#08X}", src);
```
Comment (VelocityRa) [inline src/emulator/gxm/src/gxm.cpp]: This shouldn't be here, I removed it for a reason.

File: src/emulator/gxm/src/gxm.cpp [hunk 3]
```diff
@@ -672,6 +674,7 @@ GLenum translate_internal_format(SceGxmTextureFormat src) {
             return GL_INTENSITY8;
         default:
         {
+            LOG_WARN("Unsupported translate internal format: {:#08X}", src);
```
Comment (VelocityRa) [inline src/emulator/gxm/src/gxm.cpp]: This shouldn't be here, I removed it for a reason: gxmtextlinear alread warns and with this it's smapped in the log...

**Comment by Rinnegatamante** [comment]
I've not merged your PR.

**Comment by VelocityRa** [comment]
Ok, my bad.

**Pull request merged**